compiler/luci/lang/include/luci/IR/AttrWeightsFormat.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_IR_ATTR_WEIGHTS_FORMAT_H__
+#define __LUCI_IR_ATTR_WEIGHTS_FORMAT_H__
+
+namespace luci
+{
+
+enum class WeightsFormat
```

Comment:
Seems this attribute is only used in FullyConnected and in circle schema it's `FullyConnectedOptionsWeightsFormat`
Can this be moved inside `CircleFullyConnected.h` file?

And one more step would be can this be defined inside `CircleFullyConnected` class?